tests/test_win32_console.py

```diff
@@ -37,26 +38,35 @@ def win32_handle():
         with mock.patch.object(_win32_console, "GetStdHandle", return_value=handle):
             yield handle
 
-    @patch.object(
-        _win32_console, "GetConsoleScreenBufferInfo", return_value=StubScreenBufferInfo
-    )
-    def test_cursor_position(_):
+    @pytest.fixture
+    def win32_console_getters():
+        def stub_console_cursor_info(std_handle, cursor_info):
+            cursor_info.dwSize = CURSOR_SIZE
+            cursor_info.bVisible = True
+
+        with mock.patch.object(
+            _win32_console,
+            "GetConsoleScreenBufferInfo",
+            return_value=StubScreenBufferInfo,
+        ) as GetConsoleScreenBufferInfo, mock.patch.object(
+            _win32_console, "GetConsoleCursorInfo", side_effect=stub_console_cursor_info
+        ) as GetConsoleCursorInfo:
+            yield {
```

Comment:
the fixture yields the created mocks, which is useful for tests that few need to override the default behaviour :-)